src/calendar/calendar.tsx

```diff
@@ -126,7 +126,7 @@ export default class Calendar extends Component<Props, State> {
 
   // TODO: FC Rewrite
   /* eslint-disable react/no-deprecated */
```

Comment:
Should the `eslint-disable` be removed as well then (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-deprecated.md)?

Reply:
Yes.. let's remove it since it's no longer needed